joomla/joomla-cms
# [com_fields] Add Joomla loading overlay when form submit is triggered by category selector change

Pull Request for Issue #13267 

### Summary of Changes
After comments in this PR, i have made usage of
```JS
Joomla.loadingLayer('load');
Joomla.loadingLayer('show');
```
**thus replacing the fixed message and the inline styles**,
that were originally proposed by this PR

NOTE:
-- in all cases the semi-transparent overlay and the image should shown
-- in some older browsers (IE8 / IE9) the image animation may freezing a little, but that is not a problem, it is expected

### Testing Instructions
1. Apply patch
2. Go to an article form that has custom fields
3. Change category so that form submit to display the fields assigned to newly selected category
4.  The following picture should be shown

Also same effect on **new field form**, go to fields, click new field button,
and change field type in the new field form, the same overlay should appear

![loading_please_wait](https://cloud.githubusercontent.com/assets/5092940/21466357/55bab6c4-c9d1-11e6-8eec-bed1b8c07bd0.png)

### Documentation Changes Required
none
